enhancements/installer/telemeter-metrics.md

```diff
@@ -0,0 +1,430 @@
+cluster_installation_waitfor_sum{target="cluster-complete",platform="aws",result="success",version="4.2",os="linux"} 15
+cluster_installation_waitfor_count{target="cluster-complete",platform="aws",result="success",version="4.2",os="linux"} 1
+```
+
+###### Label Values
+Potential label values are the same as the previous metric, except:
+* `le` has values between 5 and 30 minutes
+* `target` and `result` are dependent:
+  - `bootstrap-complete`
+      - `BootstrapFailed`
+      - `APIFailed`
+      - `Success`
+  - `install-complete`
+      - `OperatorsDegraded`
+      - `Success`
+
+##### cluster_installation_manifests and cluster_installation_ignition
+These metrics are simple counters totalling the number of runs for `openshift-install create manifests` or `openshift-install create ignition-configs`:
+
+```
+# HELP cluster_installation_manifests represents a single run of create manifests by the OpenShift installer.
+# TYPE cluster_installation_manifests counter
+cluster_installation_manifests{platform="aws",result="success",version="4.2",os="linux"} 1
+
+# HELP cluster_installation_ignition represents a single run of create ignition-configs by the OpenShift installer.
+# TYPE cluster_installation_ignition counter
+cluster_installation_ignition{platform="aws",result="success",version="4.2",os="linux"} 1
+```
+
+Possible `result` values are `success` and are `error`, otherwise all possible label values are the same as `cluster_installation_create`. The sample value would always be one in order to aggregate to a total count of runs of this command.
+
+##### cluster_installation_destroy
+
+When a user destroys a cluster a single sample of this metric would be pushed:
+
+```
+# HELP cluster_installation_destroy represents a single run of destroy cluster by the OpenShift installer.
+# TYPE cluster_installation_destroy histogram
+cluster_installation_destroy_bucket{platform="aws",result="success",version="4.2",os="linux",le="5"} 0
+cluster_installation_destroy_bucket{platform="aws",result="success",version="4.2",os="linux",le="10"} 1
+cluster_installation_destroy_bucket{platform="aws",result="success",version="4.2",os="linux",le="15"} 1
+cluster_installation_destroy_bucket{platform="aws",result="success",version="4.2",os="linux",le="20"} 1
+cluster_installation_destroy_bucket{platform="aws",result="success",version="4.2",os="linux",le="25"} 1
+cluster_installation_destroy_bucket{platform="aws",result="success",version="4.2",os="linux",le="30"} 1
+cluster_installation_destroy_bucket{platform="aws",result="success",version="4.2",os="linux",le="+Inf"} 1
+cluster_installation_destroy_sum{platform="aws",result="success",version="4.2",os="linux"} 10
+cluster_installation_destroy_count{platform="aws",result="success",version="4.2",os="linux"} 1
+```
+The label values are the same as `openshift_installation_create` except `le` has values between 5 and 30 minutes and `result` is `success` or `error`.
```

Comment:
Do we want to distinguish between "installer decided to error out" (e.g. we hit an error that we know we can't recover from) and "received a SIGTERM" (e.g. user got tired of waiting and gently shut us down)?